src/open_inwoner/utils/forms.py

```diff
@@ -6,6 +6,44 @@
 from django.utils.translation import gettext_lazy as _
 
 
+class ErrorMessageMixin:
+    custom_error_messages = {
+        "required": _(
+            "Het verplichte veld %s is niet (goed) ingevuld. Vul het veld in."
+        )
+    }
+
+    def __init__(self, *args, **kwargs):
+        user_added_messages = {}
+        if "custom_error_messages" in kwargs:
+            user_added_messages = kwargs.pop("custom_error_messages")
+        super().__init__(*args, **kwargs)
+        for field_name, field in self.fields.items():
```

Comment:
1. The logic in this loop is a bit hard to discern. If all you want to do is define the error message on `field` as the relevant entry from `custom_error_messages`, if it's supplied by the caller, otherwise fall back on `default_error_messages` defined on the class, it should be possible to simplify this.

2. I don't like that "required" and `required_message` are hard-coded. For every new field error that we add, we would have to update the logic as well. Try to make this more generic.